docs/netstandard-20/packaging.md

```diff
@@ -0,0 +1,259 @@
+  avoids having to update the .NET Standard package every time a platform is
+  updated or added.
+* **Compatible with .NET Standard 1.x**. The package for .NET Standard 2.0 needs
+  to work with packages that were built against .NET Standard 1.x.
+
+## Compatiblity with .NET Standard 1.x
+
+The reference assemblies for .NET Standard 1.x use a fine grained factoring.
+These assemblies are also shared with .NET Core.
+
+To represent .NET Standard 1.x we use the NuGet package `NETStandard.Library`.
+This package is a *meta package* which means it's empty, but in turn depends
+on the set of packages that make up .NET Standard. The result looks like this:
+
+    NETStandard.Library_1.3
+        ├───System.Runtime_4.1.0
+        ├───System.Security.Cryptography.Algorithms_4.2.0
+        ├───System.Security.Cryptography.Encoding_4.0.0
+        ├───System.Security.Cryptography.Primitives_4.0.0
+        ├───System.Security.Cryptography.X509Certificates_4.1.0
+        ├───System.Text.Encoding.Extensions_4.0.11
+        ├───System.Text.Encoding_4.0.11
+        ├───System.Text.RegularExpressions_4.1.0
+        ├───System.Threading.Tasks_4.0.11
+        ├───System.Threading.Timer_4.0.1
+        ├───System.Threading_4.0.11
+        ├───System.Xml.ReaderWriter_4.0.11
+        ├───...
+        └───System.Xml.XDocument_4.0.11
+
+Since we want to decouple .NET Standard from .NET Core, we decided to go with a
+different assembly set for .NET Standard 2.0, namely a monolithic assembly
+called `netstandard.dll` which contains the entirety of .NET Standard.
+
+In order to stay compatible with .NET Standard 1.x, the fine grained assemblies
+need to type forward to `netstandard.dll` when they are consumed from .NET
+Standard 2.0. Logically, this requires the following steps:
+
+1. .NET Standard 2.0 has newer versions of the fine grained assemblies that
+   type forward to `netstandard.dll`
+
+2. When consuming .NET Standard 1.x binaries from 2.0, references to the fine
+   grained assemblies are unified with the higher versions in .NET Standard 2.0
+   so that the type forwarding versions are used.
+
+We could use NuGet to model this:
+
+1. We'd have to ship updates to all the fine grained packages to incude a type
+   forwarding facade for .NET Standard 2.0.
+
+2. To ensure these higher versions are used, the package for .NET Standard 2.0
+   has to continue to depend on the fine grained packages so that it can express
+   the dependency on the higher version that has the type forwarding facade.
+
+However, we'd like to stop using these fine grained packages for both, .NET
+Standard as well as .NET Core:
+
+* **.NET Standard**. Since .NET Standard 2.0 no longer uses the fine grained
+  factoring, we don't want to continue to depend on the fine-grained
+  `System.Runtime` based packages.
+
+* **.NET Core**. While the `System.Runtime` based factoring will continue to be
+  used for .NET Core we don't expect .NET Core to ship these assemblies as
```

Comment:
Good point. I've added this section after "Layout of contract packages":

> ## Layout of out-of-band packages (e.g. `System.Collections.Immutable`)

> Components that don't have APIs that ship with any platform (i.e. are full
out-of-band) will remain as individual packages. We'll update them to target
.NET Standard so that the can be used across all .NET platforms.